inst/admiral-lookup-book.csv

```diff
@@ -1,44 +1,81 @@
 ADEG,BDS,TRT01P,Derive planned treatment for period 1,admiral,derive_vars_merged,https://pharmaverse.github.io/admiral/reference/derive_vars_merged.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
-ADEG,BDS,ADTM,Derive a datetime object from a date character ,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
-ADEG,BDS,ADTF,Derive date imputation flag,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
 ADEG,BDS,ATMF,Derive time imputation flag,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
-ADEG,BDS,ADY,Derive relative day variable,admiral,derive_vars_dy,https://pharmaverse.github.io/admiral/reference/derive_vars_dy.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
-ADEG,BDS,PARAM,Derive a list of parameter codes ,admiral,derive_vars_merged_lookup,https://pharmaverse.github.io/admiral/reference/derive_vars_merged_lookup.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
-ADEG,BDS,PARAMCD:RRR,Derive RR based on heart rate for each by group,admiral,derive_param_rr,https://pharmaverse.github.io/admiral/reference/derive_param_rr.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
-ADEG,BDS,PARAMCD:QTCBR,Derive a parameter for corrected QT using Bazett's,admiral,derive_param_qtc,https://pharmaverse.github.io/admiral/reference/derive_param_qtc.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
-ADEG,BDS,PARAMCD:QTCFR,Derive a parameter for corrected QT using Fridericia's,admiral,derive_param_qtc,https://pharmaverse.github.io/admiral/reference/derive_param_qtc.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
-ADEG,BDS,PARAMCD:QTLCR,Derive a parameter for corrected QT using Sagie's ,admiral,derive_param_qtc,https://pharmaverse.github.io/admiral/reference/derive_param_qtc.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
+ADEG,BDS,ADY,Derive analysis relative day,admiral,derive_vars_dy,https://pharmaverse.github.io/admiral/reference/derive_vars_dy.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
+ADEG,BDS,PARAM,Derive a list of parameter codes,admiral,derive_vars_merged_lookup,https://pharmaverse.github.io/admiral/reference/derive_vars_merged_lookup.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adeg.R
```

Comment:
how come this become "list of parameter"

Reply:
thanks for spotting, missed this one. This should not be PARAM, but PARAMCD. 
![image](https://github.com/user-attachments/assets/59563347-9424-4386-abff-ed714d8dfec7)